src/index.html

```diff
@@ -11,7 +11,119 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group__field">
+        <legend>Personal information</legend>
+        <label class="form__field">Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
check code style (indentaion) in all places in index.html

Reply:
I did this task

Reply:
I deleted backstop_data, but npm test don't work